charts/kamus/templates/deployment-encryptor.yaml

```diff
@@ -32,7 +32,7 @@ spec:
         release: {{ .Release.Name }}
     spec:
       serviceAccountName: {{ template "kamus.name" . }}
-      automountServiceAccountToken: false
+      automountServiceAccountToken: true
```

Comment:
Maybe make it optional?
```suggestion
      automountServiceAccountToken: {{ .Values.automountServiceAccountToken}}
```

And set the default value to false?

Reply:
That makes more sense. Is there anything else we'd need to add to the pr to register the value in the template?

Reply:
No, just add it to the readme please

Reply:
Updated the PR. Thanks